raspberry/bzzz/sensors/pressure_sensor.py

```diff
@@ -174,8 +184,9 @@ def _update_pressure(self):
             self._pressure_readings_list_current_index = 0
         self._current_pressure = median_filter(
             self._pressure_readings_list, self._median_filter_length)
-
+        
     def _calculate_altitude_from_pressure(self):
+        
         # TODO: add code to calculate altitude from pressure
```

Comment:
Is this a TODO?

Reply:
It was there in the previous code, and I may have copied that when I was copying the logic. However, I checked the code and it looks like it is calculating the altitude from the pressure reading. so I have changed the comment.